vizro-core/src/vizro/static/css/loading.css

```diff
@@ -1,22 +1,18 @@
-/* Inspired by https://github.com/facultyai/dash-bootstrap-components/blob/5c8f4b40f1100fc00bf2d5c1d671a7815a6b2910/docs/static/loading.css */
+/* Inspired by https://github.com/facultyai/dash-bootstrap-components/blob/5c8f4b40f1100fc00bf2d5c1d671a7815a6b2910/docs/static/loading.css
 
-/* This creates a dark background in situations where neither dash-loading nor the Vizro app are displayed */
+The dash-loading Div is present when Dash is initially loading, before the layout is built
+The dash-loading-callback Div is present when Dash has loaded, but the layout is still building
+
+Note that the dash-loading-callback Div is present until all elements are loaded, but as soon as the initial page
+elements (before on-page-load) are rendered, it gets pushed outside the viewable area, hence the spinner is not visible
+which is good, as we have individual loading spinners for elements. At the moment, we are not using this class.
+
+This creates a dark background in situations where neither dash-loading nor the Vizro app are displayed */
 html {
   background: rgba(20, 23, 33, 1);
```

Comment:
Is this still needed? Now the `html` should always have `data-bs-theme` which should set the background colour to be the same as the `dashboard.theme` so we should be able to remove this or at least use `background: var(...)`?

Reply:
Or maybe we don't even need the spinner here at all now? See overall comment.

Reply:
I also think that we don't need the spinner anymore and that we can stick with the coloured background only.

Reply:
I removed it 👍